dask/dataframe/io/tests/test_parquet.py

```diff
@@ -64,11 +64,16 @@
 SKIP_PYARROW_LE_REASON = "pyarrow not found"
 SKIP_PYARROW_DS = SKIP_PYARROW
 SKIP_PYARROW_DS_REASON = "pyarrow not found"
-if pa and pa_version.major >= 5:
-    SKIP_PYARROW_LE = True
-    SKIP_PYARROW_LE_REASON = "pyarrow < 5.0.0 required for pyarrow legacy API"
-PYARROW_LE_MARK = pytest.mark.skipif(SKIP_PYARROW_LE, reason=SKIP_PYARROW_LE_REASON)
-
+if pa_version.major >= 5 and not SKIP_PYARROW:
+    # NOTE: We should use PYARROW_LE_MARK to skip
+    # pyarrow-legacy tests once pyarrow officially
+    # removes ParquetDataset support in the future.
+    PYARROW_LE_MARK = pytest.mark.filterwarnings(
+        "ignore::DeprecationWarning",
+        "ignore::FutureWarning",
```

Comment:
```suggestion
        "ignore:'ParquetDataset:DeprecationWarning"
```

Could also make it more strict (to only filter out those specific warnings). I think this should work right now, but might also be a bit brittle of course.

Reply:
Not sure if I am doing something wrong, but pytest fails to filter the pyarrow warnings when when I do `"ignore:'ParquetDataset:DeprecationWarning"` instead of `"ignore::DeprecationWarning"`.  Also, it seems `"ignore::FutureWarning"` is required to filter the warnings in Dask.

Reply:
>  pytest fails to filter the pyarrow warnings when when I do

Hmm, it works in the pyarrow tests .. Anyway, probably not worth trying to figure out why, it was just a minor point!

>  Also, it seems "ignore::FutureWarning" is required to filter the warnings in Dask.

Ah, yes, that's a good point